helm-chart/splunk-kubernetes-metrics/values.yaml

```diff
@@ -81,13 +82,21 @@ secret:
 
 
 image:
-  name: splunk/k8s-metrics:1.1.1
+  registry: docker.io
+  name: splunk/k8s-metrics
+  tag: 1.1.0
```

Comment:
tag is 1.1.1

Reply:
did you want the tags for all of the images set to 1.1.1 or just for metrics?

Reply:
just for metrics :)